sync.go

```diff
@@ -9,17 +9,14 @@ import (
 	"time"
 
 	"github.com/decred/dcrd/addrmgr"
-	"github.com/decred/dcrd/rpcclient"
-	"github.com/decred/dcrwallet/chain"
-	"github.com/decred/dcrwallet/errors"
-	"github.com/decred/dcrwallet/p2p"
-	"github.com/decred/dcrwallet/spv"
-	"github.com/decred/dcrwallet/wallet"
+	"github.com/decred/dcrwallet/errors/v2"
+	p2p "github.com/decred/dcrwallet/p2p/v2"
+	spv "github.com/decred/dcrwallet/spv/v3"
+	wallet "github.com/decred/dcrwallet/wallet/v3"
```

Comment:
Since we've renamed the `spv` module, can we use `github.com/raedahgroup/dcrlibwallet/spv` here? Then we can remove the replace directive in go.mod.